sources/cfsignal/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "cfsignal"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[dependencies]
+log = "0.4"
+serde = { version = "1.0", features = ["derive"] }
+simplelog = "0.11"
+snafu = { version = "0.7" }
+toml = "0.5.1"
+tokio = { version = "~1.14", default-features = false, features = ["macros", "rt-multi-thread"] }
+rusoto_core = { version = "0.47.0", default-features = false, features = ["rustls"] }
+rusoto_cloudformation = { version = "0.47.0", default-features = false, features = ["rustls"] }
```

Comment:
Should we switch to the AWS SDK now instead of later?

Reply:
I would rather do the switch all at once instead of having to build crates for two different SDKs.

Reply:
https://github.com/bottlerocket-os/bottlerocket/issues/1581#issuecomment-909325897